applicationinsights/TelemetryClient.py

```diff
@@ -200,7 +200,18 @@ def track_request(self, name, url, success, start_time=None, duration=None, resp
         data.id = str(uuid.uuid4())
         data.name = name
         data.start_time = start_time or datetime.datetime.utcnow().isoformat() + 'Z'
-        data.duration = duration or 0
+
+        local_duration = duration or 0
+        duration_parts = []
+        for multiplier in [1000, 60, 60, 24]:
+            duration_parts.append(local_duration % multiplier)
+            local_duration = int(local_duration / multiplier)
```

Comment:
This can be `local_duration //= multiplier` (`//` always truncates to int - `/` only truncates on Python 2)